package.json

```
@@ -27,7 +27,7 @@
   },
   "packageManager": "yarn@1.22.19",
   "engines": {
-    "node": "^18.19.0"
+    "node": ">=18.19.0"
```

sorry I haven't updated this since I just tested Nodejs's compatibility.
https://www.electronjs.org/docs/latest/tutorial/electron-timelines

```suggestion
    "node": ">=20.14.0"
```